modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-an-variable.adoc

```diff
@@ -76,7 +61,6 @@ data:
   mykey: myvalue
   otherkey: othervalue
 ----
-+
-This results in two environment variables with names `FOO_ENV`, `OTHER_ENV`, and values `myvalue` and `othervalue`, being provisioned into all workspace containers.
-+
-NOTE: The maximum length of annotation names in a {orch-name} secret is 63 characters, where 9 characters are reserved for a prefix that ends with `/`. This acts as a restriction for the maximum length of the key that can be used for the secret.
+
+* The maximum length of an annotation name in a {orch-name} secret is 63 characters, of which 9 characters are reserved for a prefix that ends with `/`. This acts as a restriction for the maximum length of the key that can be used for the secret.
+//"annotation name" is referring to `name` or `env-name`? This is not clear from the code snippets presented in the code blocks here and so needs to be clarified. max-cx
```

Comment:
Please, clarify this and modify the content accordingly.

Reply:
@rkratky I've saved this comment in a new topic branch for the follow-on issue (3101), so I will work on it there. The lack of clarity is in the original text, and I did not change its meaning; hence, it can be left in the text as is for now. I have also posted this question in #forum-devtools-docs and will ask someone again later. So here I am simply removing this comment line from this PR, which can be merged then.